nimbleparse/src/main.rs

```diff
@@ -159,6 +159,11 @@ fn main() {
         }
     };
 
+    if let Some(c) = stable.conflicts() {
+        println!("Stategraph:\n{}\n", sgraph.pp_core_states(&grm));
```

Comment:
Should we say something like "Conflicts found: {} shift/reduce {} reduce/reduce\n  {}" here? Otherwise the user might not realise what the stategraph is for?

Reply:
Hmm, I think the next line already says something like
```
Shift/Reduce conflicts:
   State: 1: Shift(2) / Reduce(E: T)
```

So we'd mention "Shift/Reduce conflicts" twice. Are you still fine with that?

Reply:
Or maybe we just swap the lines, and put the conflicts before the graph?

Reply:
Swapping the lines works for me!